Cargo.toml

```diff
@@ -24,6 +24,10 @@ symbolic-common = "7.2"
 symbolic-debuginfo = "7.2"
 symbolic-demangle = "7.2"
 symbolic-minidump = "7.2"
+#symbolic-common = { path = "../symbolic.calixteman/common" }
+#symbolic-debuginfo = { path = "../symbolic.calixteman/debuginfo" }
+#symbolic-demangle = { path = "../symbolic.calixteman/demangle" }
+#symbolic-minidump = { path = "../symbolic.calixteman/minidump" }
```

Comment:
?

Reply:
Some of Calixte's PRs haven't been merged upstream yet. IIRC the changes are about how lines are handled (for inlined functions and the like). Originally we though we'd write that functionality in **dump_syms** like we currently have in the Breakpad version but the Sentry team expressed interest of having that into Symbolic and so Calixte is doing the work upstream.